recipe/meta.yaml

```
@@ -14,14 +14,14 @@ source:
   sha256: cc5ce27aa30b5ade54d520fafc1cd7d293dfba58c51851759a034d03e3f370bb
 
 build:
-  number: 3
+  number: 4
   skip: True  # [osx or py2k or (win and vc<14)]
```

```suggestion
  skip: True  # [osx or win]
```

* python 2 was dropped from conda forge.
* vc<14 was dropped from conda-forge

windows does not have pytorch builds on conda-forge